king_phisher/client/dialogs/configuration.py

```diff
@@ -265,6 +269,7 @@ def interact(self):
 		response = self.dialog.run()
```

Comment:
It doesn't look like you have the logic to set the applicable GTK entries. Before `dialog.run()` you should parse the URL, and then populate the URL, username and password GTK entry widgets.